Simplenote/SPAppDelegate.m

```diff
@@ -562,7 +562,7 @@ - (void)bucket:(SPBucket *)bucket didReceiveObjectForKey:(NSString *)key version
 {
     if ([bucket.name isEqualToString:@"Note"]) {
         if ([key isEqualToString:self.noteEditorViewController.currentNote.simperiumKey]) {
-            [self.noteEditorViewController didReceiveVersion:version data:data];
+            [self.noteEditorViewController.historyLoader processData:data forVersion:[version integerValue]];
```

Comment:
Problem with this approach is: iPad + Multiple Windows (not supported yet, but it'd be good if we could adjust things up a bit!).

Please take a look at [VersionsController](https://github.com/Automattic/simplenote-macos/blob/develop/Simplenote/VersionsController.swift) in the Mac project, that'd help us convert this into a Publisher / Subscriber (which should work with N editor instances).

Reply:
We will probably need to adjust VersionController as well to support multiple subscribers for the same simperium entity.